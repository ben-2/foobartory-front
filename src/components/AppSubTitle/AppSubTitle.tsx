import styles from './AppSubTitle.module.css';

export function AppSubTitle() {
  return (
    <>
      <div className={styles.subtitle}>
        Cha√Æne de production
      </div>
    </>
  );
}
