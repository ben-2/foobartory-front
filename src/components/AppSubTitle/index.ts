import { AppSubTitle } from "./AppSubTitle";
export default AppSubTitle;