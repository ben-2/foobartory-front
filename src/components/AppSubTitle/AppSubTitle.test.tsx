import { AppSubTitle } from './AppSubTitle';
import { shallow } from 'enzyme';

describe('<AppTitle />', () => {
  it('renders Chaine de production automatisee subtitle', () => {
    const wrapper = shallow(<AppSubTitle />)
    expect(wrapper).toMatchSnapshot();
  })
});
