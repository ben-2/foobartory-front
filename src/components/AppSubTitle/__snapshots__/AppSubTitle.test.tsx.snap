// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AppTitle /> renders Chaine de production automatisee subtitle 1`] = `ShallowWrapper {}`;
